docs/rest/src/files/CheckFileInfo.rst

```diff
@@ -591,6 +591,19 @@ Other miscellaneous properties
         ..  admonition:: |wac| Tip
 
             See :ref:`View performance` for more details on how this property is used in |wac|.
+			
+    SharingStatus
+        |prerelease|
```

Comment:
What is the client supposed to do with these values? What is it _not_ supposed to do with the values? What if they're omitted?

Reply:
The client in this case being the WOPI client.